tilelang/language/kernel.py

```
@@ -309,6 +315,17 @@ def Kernel(
     if prelude is not None:
         attrs["pragma_import_c"] = prelude
 
+    if cluster_dims is not None:
+        if isinstance(cluster_dims, (list, tuple)):
+            cluster_dims = list(cluster_dims) + [1] * (3 - len(cluster_dims))
+        elif isinstance(cluster_dims, int):
+            cluster_dims = [cluster_dims, 1, 1]
+        else:
+            raise ValueError("cluster_dims must be a list or tuple of integers")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)